### PR TITLE
[sqlserver] Fix timestamp type parse of sqlserver

### PR DESCRIPTION
**Pull request by loserwang1024**
Fix https://github.com/ververica/flink-cdc-connectors/issues/2637

### CODE REVIEW

**Comment by loserwang1024** [comment]
Hi, @GOODBOY008 , I have modified how to parse timestam_ltz value in testTemporalTypesWithTimeZone. 
However, in com.ververica.cdc.connectors.sqlserver.table.SqlServerTimezoneITCase#testTemporalTypesWithTimeZone, I am wondering why you get different TIMESTAMP_LTZ values(UTC time) in different timezone ?

Do you mean that TO_TIMESTAMP(val_datetimeoffset) ? A TIMESTAMP_LTZ value can be parsed to different TIMESTAMP values in in different timezone.

**Comment by GOODBOY008** [comment]
@loserwang1024 Great catch, Can you combine the unit tests and fix code format?

**Review by GOODBOY008 — COMMENTED** [review]
@loserwang1024 Great catch, Can you fix code format?

File: flink-cdc-connect/flink-cdc-source-connectors/flink-connector-sqlserver-cdc/src/test/java/com/ververica/cdc/connectors/sqlserver/table/SqlServerTimezoneITCase.java [hunk 1]
```diff
@@ -106,35 +106,178 @@ public void testTemporalTypesWithTimeZone() throws Exception {
...
+                        "+I[0, 2018-07-13, 10:23:45.680, 10:23:45.678, 2018-07-13T11:23:45.340, 2018-07-13T01:23:45.456Z, 2018-07-13T13:23:45.780, 2018-07-13T14:24]");
+        ;
+        List<String> actual = TestValuesTableFactory.getRawResults("sink");
+        assertEquals(expected, actual);
+
+        result.getJobClient().get().cancel().get();
+    }
+
+    @Test
+    public void testTimeZone_ltz2ntz() throws Exception {
```
Comment (GOODBOY008) [inline flink-cdc-connect/flink-cdc-source-connectors/flink-connector-sqlserver-cdc/src/test/java/com/ververica/cdc/connectors/sqlserver/table/SqlServerTimezoneITCase.java:130]: Merge unit tests.

**Comment by loserwang1024** [comment]
@GOODBOY008 ， 
``` java
TimestampData.fromLocalDateTime(
                            LocalDateTime.ofInstant(instant, serverTimeZone));
```
@GOODBOY008 , LocalDateTime.ofInstant(instant, serverTimeZone)) use serverTimeZone, while TimestampData.fromLocalDateTime use JVM's default timezone. If this two timezones are not same, time will change.

And I design a test with different  timezones:
* testTimeZoneWithDifferentServerTimeZones
* testTimeZoneWithDifferentLocalTimeZones
* [testTimeZoneWithDifferentJVMTimeZones](https://github.com/loserwang1024/flink-cdc-connectors/blob/9b3b6f914052699a0aece7fefaa1ab4905b11d41/flink-cdc-connect/flink-cdc-source-connectors/flink-connector-sqlserver-cdc/src/test/java/com/ververica/cdc/connectors/sqlserver/table/SqlServerTimezoneITCase.java#L86)

By the way, MSSQL_SERVER_CONTAINER can not run on my m2 Macbook, it's hard to develop server-sql cdc.

**Review by GOODBOY008 — APPROVED** [review]
@loserwang1024 Great catch , Thanks for your contribution. LGTM

**Pull request merged**